src/components/GasMarketWidget.tsx

```diff
@@ -38,8 +38,12 @@ const formatGasTooltip = (
 ${gasStr} Gwei`;
 };
 
+const getBlockPageLink = (u: unknown): string | undefined =>
```

Comment:
Could you narrow down the argument type here? `unknown` is very wide. It suggests `null`, and `"notablocknumber"` are fine arguments, and although the function wouldn't blow up, it would produce a broken link.

A different way to think about this is: you've written a function that only works if you pass and argument that is a number, or a string that is actually a valid number. The second is kinda confusing and we don't use it, so let's just say the function is written to handle numbers only.

Then `u` can be unknown but the function should be:
```ts
typeof u === "number" ? `https://...${u}` : undefined
```

or `u` should be of type `u: number | undefined` and then the following is fine, because we're again only handling numbers.
```ts
typeof u === undefined ? undefined : `https://...${u}`
```

Am I explaining that well?

Reply:
Got it! I found this code snippet in BurnRecord. Do you want me to change it there too?
![image](https://user-images.githubusercontent.com/48362877/211286515-3583dfa5-b72d-49c7-8716-4529582054ed.png)

Reply:
Oh, yes please!